assets/src/edit-story/components/canvas/gridview/index.js

```diff
@@ -58,36 +58,67 @@ const RangeInputWrapper = styled.div`
   margin: 0 auto 75px auto;
 `;
 
-const Rectangle = styled.div`
+const Rectangle = styled.button.attrs(({ isDisabled }) => ({
+  disabled: isDisabled,
```

Comment:
Why rename this? Why not just use `disabled` [in the component](https://github.com/google/web-stories-wp/pull/345/files#diff-8b72df94c15920b4985161f9112c8382R104)?

Reply:
Because it is more makes sense readable. Of course I took some examples of this applications following the coding style and naming rule ;)

Reply:
Sure, I can see that. But for native properties in HTML, I'd prefer to just use those – though some of them are a bit succinctly named.

Reply:
OK. Done